src/esp/sim/Simulator.cpp

```diff
@@ -890,12 +885,6 @@ double Simulator::getPhysicsTimeStep() {
 bool Simulator::recomputeNavMesh(nav::PathFinder& pathfinder,
                                  const nav::NavMeshSettings& navMeshSettings,
                                  const bool includeStaticObjects) {
-  ESP_CHECK(config_.createRenderer,
-            "::recomputeNavMesh: "
-            "SimulatorConfiguration::createRenderer is false. Scene "
-            "geometry is required to recompute navmesh. No geometry is "
-            "loaded without renderer initialization.");
-
```

Comment:
I removed this assert as it is obsolete.
`ResourceManager` already imports scene geometry to process physics whether or not the renderer is present.